README.md

```diff
@@ -128,22 +167,41 @@ remote_write:
         action: drop
 ```
 
-Additional information about setting up relabel configs, the `source_labels` field, and the possible actions can be found in the [Prometheus Docs](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write).
+Additional information about setting up relabel configs, the
+`source_labels` field, and the possible actions can be found in the
+[Prometheus Docs](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write).
 
-### Configuring the Timescale-Prometheus Connector Binary
+#### Configuring the Timescale-Prometheus Connector Binary
 
-The Timescale-Prometheus Connector Binary is configured either through CLI flags or environment variables.
-The list of all available flags is displayed on the help `timescale-prometheus -h` command. All
+The Timescale-Prometheus Connector Binary is configured either through
+CLI flags or environment variables. The list of all available flags is
+displayed on the help `timescale-prometheus -h` command. All
 environment variables are prefixed with `TS_PROM`.
 
-## Building
+### Binaries
```

Comment:
I think this and the following 2 sections belong under "Non-Helm installation methods" These do not matter for helm. Not sure why they were moved.

Reply:
Is it just the "Binaries" and "Docker" sections? Because "Building from source" was a higher-level section before.

Reply:
Note after chatting with Mat: Keep "Building from source" separate. Put all 3 sections under non-helm.